### PR TITLE
Add Crystal 1.4.0 and 1.3.2

### PR DESCRIPTION
**Pull request by HertzDevil**
Related: compiler-explorer/infra#701

### CODE REVIEW

File: etc/config/crystal.amazon.properties [hunk 1]
```diff
@@ -10,9 +10,13 @@ group.crystal.supportsExecute=true
 group.crystal.compilerType=crystal
 group.crystal.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 
-compiler.crystal131.semver=1.3.1
```
Comment (mattgodbolt) [inline etc/config/crystal.amazon.properties:13]: Please leave the old compilers in. We don't remove compilers (else folks who've bookmarked things will get errors).

Comment (mattgodbolt) [reply etc/config/crystal.amazon.properties:13]: I'm so sorry- @RubenRBS has reminded me and pointed out that you're aliasing here :blush: All's well; thanks!

**Review by mattgodbolt — APPROVED** [review]

File: etc/config/crystal.amazon.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 compilers=&crystal
 defaultCompiler=crystal131
```
Comment (mattgodbolt) [inline etc/config/crystal.amazon.properties:2]: @HertzDevil can you fix the default compiler please

```
Found 1 property file(s) with mismatching ids:
./etc/config/crystal.amazon.properties
UNKNOWN DEFAULT COMPILER:
	crystal131
```

**Review by mattgodbolt — CHANGES_REQUESTED** [review]

File: etc/config/crystal.amazon.properties (hunk 2 above)
Comment (RubenRBS) [reply etc/config/crystal.amazon.properties:2]: +1 for the script catching this, good thing it's there!

**Comment by mattgodbolt** [comment]
Thanks!

**Comment by mattgodbolt** [comment]
This is now live!

**Pull request merged**